app/blog/2020-09-22-lit-element-and-lit-html-next-preview.md

```diff
@@ -0,0 +1,57 @@
+
+   We also want to make the directive API SSR compatible by limiting access to the DOM.
+
+## What's changed
+
+These are new major versions, so there are some breaking changes, but we want to minimize disruption to our users as much as possible, so we've limited the breaking changes so that they don't affect most users, or only require a mechanical change (like changing an import).
+
+Please see the READMEs ([LitElement](https://github.com/Polymer/lit-html/blob/lit-next/packages/lit-element/README.md), [lit-html]((https://github.com/Polymer/lit-html/blob/lit-next/packages/lit-html/README.md))) and CHANGELOGs ([LitElement](https://github.com/Polymer/lit-html/blob/lit-next/packages/lit-element/CHANGELOG.md), [lit-html]((https://github.com/Polymer/lit-html/blob/lit-next/packages/lit-html/CHANGELOG.md))) for the most detailed list of changes.
+
+The most important changes:
+
+  * Customizing the syntax of lit-html is no longer directly supported. The `templateFactory` and `TemplateProcessor` APIs are removed.
+  * The public API has been minimized in order to facilitate better minification and future evolution.
+  * The lit-html directive has changed to be class-based and to persist directive instances. Directives should be easier to write and easier to make SSR compatible.
+  * The LitElement decorators are no-longer exported from the main module - they have to be imported individually or from a new `lit-element/decorators.js` module. This means smaller app sizes for non-decorator-users and opens the door to new decorators implementing the current TC39 JavaScript proposal when those arrive.
+  * lit-html no longer uses `instanceof` or module-level WeakMaps to detect special objects like template results and directives, which should improve the compatibility of multiple copies of lit-html in a single app. We still reccomend de-duping npm packages, but more cases will work now.
+  * Safari 12 has a critical [template literal bug](https://bugs.webkit.org/show_bug.cgi?id=190756), which is no longer worked around in lit-html. If you support Safari 12 you will have to compile template literals to their ES5 equivalent. Note that babel-preset-env already does this for the broken versions of Safari.
+
+There are smaller changes listed in the change logs. Overall we hope these versions are drop-in replacements for most users, or only require updating decorator imports.
+
+## Installation
+
+Run:
+
+```sh
+npm i lit-element@next-major
+```
+and/or
+```sh
+npm i lit-html@next-major
+```
+
+## Submitting Feedback
+
+LitElement has moved into the lit-html repo, which we are converting to a mono-repo. Please file issues with a prefix of `[lit-html]` or `[lit-element]`.
```

Comment:
I feel like this wording could give the wrong impression (i.e., that we're de-emphasizing LitElement in favor of lit-html). 

Although it's kind of overkill, maybe:

We're in the process of moving the next versions of LitElement and lit-html into a mono-repo. Please file issues on the current lit-html repo, using a prefix of `[lit-html]` or `[lit-element]`.

... possibly with links to issue templates for those issues.

Reply:
Or for now maybe just link "current lit-html repo" to https://github.com/Polymer/lit-html/issues/new/choose

Reply:
did both